rights-statements.ttl

```diff
@@ -28,202 +30,219 @@
 
 <collection-ic> a skos:Collection ;
   skos:prefLabel "rights statements for works that are in copyright"@en ;
-  skos:member <InC> ;
-  skos:member <InC-OW-EU> ;
-  skos:member <InC-RUU> ;
-  skos:member <InC-EDU> ;
-  skos:member <InC-NC> .
+  skos:member <InC/1.0/> ;
+  skos:member <InC-OW-EU/1.0/> ;
+  skos:member <InC-RUU/1.0/> ;
+  skos:member <InC-EDU/1.0/> ;
+  skos:member <InC-NC/1.0/> .
 
 <collection-pd> a skos:Collection ;
```

Comment:
Should the collection URIs be versioned? I think they at least need trailing slashes.